stable_baselines3/common/env_checker.py

```diff
@@ -266,14 +266,19 @@ def check_env(env: gym.Env, warn: bool = True, skip_render_check: bool = True) -
         # Check for the action space, it may lead to hard-to-debug issues
         if isinstance(action_space, spaces.Box) and (
             np.any(np.abs(action_space.low) != np.abs(action_space.high))
-            or np.any(np.abs(action_space.low) > 1)
-            or np.any(np.abs(action_space.high) > 1)
+            or np.any(action_space.low != -1)
```

Comment:
Hmm any pointer/argument for also complaining about smaller intervals than [-1, 1]? I guess it is easier to give an exact suggestion for the normalization, though :)

Reply:
same pointer as usual (already in the warning): https://stable-baselines3.readthedocs.io/en/master/guide/rl_tips.html
"Why should I normalize the action space?" section

Reply:
Right, I am just more wondering the direction of "do not go below [-1, 1]" (the Gaussian normal thing, for example, covers more range than [-1, 1]). In any case, it makes sense to warn about this to avoid having people going "too small" :). Resolved 👍

Reply:
https://stable-baselines3.readthedocs.io/en/master/_images/mistake.png

range too small: the gaussian always sample the limits (gaussian is clipped).